flexbox/src/androidTest/java/com/google/android/flexbox/test/TestAdapter.java

```diff
@@ -67,6 +76,14 @@ void addItem(FlexboxLayoutManager.LayoutParams flexItem) {
         notifyItemInserted(mLayoutParams.size() - 1);
     }
 
+    void changeItemWithPayload(int position, Object payload) {
+        notifyItemChanged(position, payload);
+    }
+
+    List<Object> getPayloads() {
+        return Collections.unmodifiableList(mReceivedPayloads);
```

Comment:
This seems to be API level 24. Is it alright?
https://developer.android.com/reference/java/util/Collections.html#unmodifiableList(java.util.List<? extends T>)